### PR TITLE
Create CODEOWNERS file

### PR DESCRIPTION
**Pull request by gdbelvin**
A CODEOWERS file will allow github to automatically assign reviewers to PRs so
incoming PRs from the community don't get lost.

A CODEOWNERS file will also require an LGTM from one of the indicated owners
before PRs get merged.  

This could help ensure that the right people sign off on PRs and make it clear
who the appropriate stakeholders are.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+# Default
+*                          @google/certificate-transparency
+*.proto                    @Martin2112 @daviddrysdale
```
Comment (Martin2112) [inline CODEOWNERS]: +Al maybe.

File: CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+# Default
+*                          @google/certificate-transparency
```
Comment (Martin2112) [inline CODEOWNERS]: Shouldn't that be this repo, not the C++ one?

Comment (daviddrysdale) [reply CODEOWNERS]: I think that's a [team](https://github.com/orgs/google/teams/certificate-transparency/members) not a repo (confusingly)

Comment (Martin2112) [reply CODEOWNERS]: Ah, it confused me. Never mind then.

**Comment by Martin2112** [comment]
I assume it doesn't have any sort of OOO integration anywhere?

File: CODEOWNERS (hunk 1 above)
Comment (gdbelvin) [reply CODEOWNERS]: Done. @AlCutter Are you ok with that?

**Pull request merged**